backend/src/graphql/arg/UnsecureLoginArgs.ts

```diff
@@ -7,4 +7,7 @@ export default class UnsecureLoginArgs {
 
   @Field(() => String)
   password: string
+
+  @Field(() => Boolean)
+  hasElopage?: boolean
```

Comment:
Same comment as in UserResolver, I think that this should be a new response parameter.